lighthouse-core/audits/accessibility/bypass.js

```diff
@@ -12,6 +12,20 @@
  */
 
 const AxeAudit = require('./axe-audit');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if the page has elements that let screen reader users skip over repetitive content. This descriptive title is shown to users when no user action is required. */
+  title: 'The page contains a heading, skip link, or landmark region',
```

Comment:
Do we need to put the technical terms in quotes e.g. `..contains a 'heading', 'skip link', or 'landmark region'.`  So that there is no confusion in translation, or call out that they are technical terms that cannot be broken up?

Reply:
Yeah should probably call this out in description 👍